### PR TITLE
Documentation improved

### PR DESCRIPTION
**Pull request by susliko**
This pull request introduces documentation for `tofu-config` and brings some changes to existing pages:
* common naming style for names of pages on the side panel
* installation notes in the beginning of each page, dedicated to a specific module
* optics hierarchy diagram is now displayed
* `errors.md` page reworked
* `Gettings Started` reworked
* alphabetical sorting for imports in examples

### CODE REVIEW

File: docs/optics.md [hunk 1]
```diff
@@ -3,20 +3,28 @@ id: optics
...
 Hierarchy
 ---------
 
-[![Optics Hierarchy](./optics-hierarchy.png)]
+<img src="/tofu/docs_img/optics-hierarchy.png" height="700">
 
 Intuition
 ---------
 
-If You ever used [lens](https://github.com/ekmett/lens) optics' names will be quite familiar to You,
```
Comment (danslapman) [inline docs/optics.md:18]: Pronouns are written from the capital letter in english, this must be reverted

File: docs/optics.md [hunk 2]
```diff
@@ -3,20 +3,28 @@ id: optics
...
-but for a [Monocle](https://github.com/julien-truffaut/Monocle) user Tofu's naming may be confusing. The following table may clear things up:
+If you ever used [lens](https://github.com/ekmett/lens) optics' names will be quite familiar to you,
```
Comment (danslapman) [inline docs/optics.md:26]: Pronouns are written from the capital letter in english, this must be reverted

Comment (susliko) [reply docs/optics.md:26]: https://www.dictionary.com/e/when-to-capitalize-words/
"Pronouns are words that replace nouns. I, you, and me are all examples of pronouns. While you and me are usually lowercase, the pronoun I should always be capitalized, regardless of where it appears in a sentence."

Comment (danslapman) [reply docs/optics.md:26]: Roger that

**Pull request merged**